README.md

```diff
@@ -12,15 +14,39 @@ fastlane add_plugin run_tests_firebase_testlab
 
 ## About run_tests_firebase_testlab
 
+Authenticates with Google Cloud.
 Runs Android tests in Firebase testlab.
-
-**Note to author:** Add a more detailed description about this plugin here. If your plugin contains multiple actions, make sure to mention them here.
+Fetches the results to a local directory.
+Deletes the results from firebase bucket if wanted.
 
 ## Example
 
-Check out the [example `Fastfile`](fastlane/Fastfile) to see how to use this plugin. Try it by cloning the repo, running `fastlane install_plugins` and `bundle exec fastlane test`.
+Check out the [example `Fastfile`](fastlane/Fastfile) to see how to use this plugin.
+
+### Simple usage
+
+```
+run_tests_firebase_testlab(
+    project_id: "your-firebase-project-id",
+    model: "Nexus6P",
+    version: "27")
+```
 
-**Note to author:** Please set up a sample project to make it easy for users to explore what your plugin does. Provide everything that is necessary to try out the plugin in this project (including a sample Xcode/Android project if necessary)
+### Parameters
+
+- `project_id`: Your Firebase project id. This parameter is necessary.
+- `model`: The device model name to run the tests. This parameter is necessary.
+- `version`: The Android api version of the device. This parameter is necessary.
+- `app_apk`: The path for your app apk. Default: app/build/outputs/apk/debug/app-debug.apk
+- `android_test_apk`: The path for your android test apk. Default: app/build/outputs/apk/androidTest/debug/app-debug-androidTest.apk
```

Comment:
Should we have a table for the default values of each param instead of putting it like this? I'm not sure what is the usual approach.

Reply:
Checkout how they style the Swagger Configuration options in https://github.com/richhollis/swagger-docs